### PR TITLE
(fleet/dashboards) fix some dashboards and add new ones

### PR DESCRIPTION
**Pull request by fbegyn**
The grafana dashboards are using specific datasources in the first design. This causes the UUID to break when importing these through provisioning files into other grafana instances. The fix for this is to use a datasource variable.

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Comments including screenshots are in this thread on slack: https://lsstc.slack.com/archives/C05JLDDAPAR/p1715360161601749

**Comment by jhoblitt** [comment]
The commit message also contains a typo: "dashboars"

**Comment by fbegyn** [comment]
I've cleaned up a lot of dashboards to bring them back into working state I believe. The only 2 broken parts are the `PDU montoring` and `Base to summit temporary`. I'd prefer to get this PR merged in the current state, further tweaks to those 2 and other dashboards (improving network overview) can be done in subsequent PRs.

File: fleet/lib/grafana-dashboards/dashboards/hosts-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/hosts-overview.json:1]: The `mds_hosts` and `rgw_hosts` drop downs don't have any values.

image

File: fleet/lib/grafana-dashboards/dashboards/osd-device-details.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/osd-device-details.json:1]: The physical device metrics aren't working for me on ruka and I don't understand why.  The metrics are present in prometheus. 
image
image

File: fleet/lib/grafana-dashboards/dashboards/osds-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/osds-overview.json:1]: I have to scale the fontsizes way down to get remove horizontal scroll bars from the latency tables.

image

At 60% the values become visible:

image

File: fleet/lib/grafana-dashboards/dashboards/pool-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/pool-overview.json:1]: The pool overview table is is hard to use.  There are a lot of columns of duplicated or unimportant data that can be removed. The important information is on the right side of the table.  The vertical height should be increased as we have a few clusters with a fair number of pools.

image

File: fleet/lib/grafana-dashboards/dashboards/radosgw-detail.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/radosgw-detail.json:1]: This dashboard appears to be broken on both ruka and ayekan.

image

File: fleet/lib/grafana-dashboards/dashboards/radosgw-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/radosgw-overview.json:1]: This dashboard seems to be broken on ruka and ayekan for all clusters.

image

File: fleet/lib/grafana-dashboards/dashboards/radosgw-sync-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/radosgw-sync-overview.json:1]: We don't currently have rgw zone replication enabled on any cluster. We have tested it in the past and will probably use it later this year.  It currently isn't possible to test with dashboard.

File: fleet/lib/grafana-dashboards/dashboards/rbd-details.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/rbd-details.json:1]: This dashboard seems to be broken on both ruka and ayekan.

image

File: fleet/lib/grafana-dashboards/dashboards/rbd-overview.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/rbd-overview.json:1]: The tables on this dashboard are difficult to read with my browser fonts at 100%.  I have to scale it down to 60% to get all the data to render without horizontal scroll bars.

100%:
image

60%:
image

File: fleet/lib/grafana-dashboards/dashboards/rgw-s3-analytics.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/rgw-s3-analytics.json:1]: This dashboard seems to be broken on both ruka and ayekan for all cluster / rgw_server combos I tried.

image

File: fleet/lib/grafana-dashboards/dashboards/rook-ceph.json
Comment (jhoblitt) [inline fleet/lib/grafana-dashboards/dashboards/rook-ceph.json:1]: Is this dashboard a duplicate of `Ceph - Cluster`?  It does not support multiple clusters.

image

**Review by jhoblitt — CHANGES_REQUESTED** [review]
I have completed inspect all dashboard within this PR and made comments where I have concerns. WRT the Ceph dashboards.  If the titles are direct copies of upstream dashboards, I think its OK to not rename them to have "Ceph" in the title but I think they need to be in a Ceph (or Rook) folder to logically group them together.

**Comment by jhoblitt** [comment]
@KrisBuytaert @fbegyn What is the status of this PR?  Is it ready for re-review?

**Comment by jhoblitt** [comment]
I'm going to go ahead and merge this as-is. Any remaining issues will be addressed in new PRs.

**Pull request merged**